deploy/firesim

```diff
@@ -3,7 +3,7 @@
 
 # REQUIRES PYTHON2, because fabric requires python2
 
-from __future__ import with_statement, print_function
+from __future__ import with_statement
```

Comment:
oops, I thought I added print_function as a part of this here.  I could put it back but I don't think it's used.